public/index.html

```diff
@@ -62,8 +54,7 @@ <h1 class='h1'>Twin Cities Aid Map</h1>
   <div class='map' id="map"></div>
 
   <script src="//cdnjs.cloudflare.com/ajax/libs/list.js/1.5.0/list.min.js"></script>
```

Comment:
Can't say I'm particularly enamored with snowplow, but I'm not sure this would have been any easier with webpack. List.js is just... weird.

Reply:
Given that this is also listed in the deps of package.json...what's going on here?

Reply:
Yeah. Can you elaborate on what's going in here?

Reply:
I changed this so that it points at the vendor directory `web_modules` instead of the CDN. I'm kinda torn as to whether it's better to drop it from the vendors and use it in the CDN instead. On one hand it's an easier upgrade story, and I like the idea of controlling versions. On the other... it's a little odd.